specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```diff
@@ -0,0 +1,1443 @@
+        }
+      }
+    },
+    "PaginatedInferenceEndpointDeploymentsList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning Inference Deployment objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array of Machine Learning Inference Deployment  objects wrapped in ARM resource envelope.",
+          "items": {
+            "$ref": "#/definitions/InferenceEndpointDeploymentResource"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A continuation link (absolute URI) to the next page of results in the list."
+        }
+      }
+    },
+    "SystemData": {
+      "description": "Specifies system data of the resource.",
+      "properties": {
+        "createdBy": {
+          "type": "string"
+        },
+        "createdByType": {
+          "type": "string",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "UserTypes",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string"
+        },
+        "lastModifiedBy": {
+          "type": "string"
+        },
+        "lastModifiedByType": {
+          "type": "string",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "UserTypes",
+            "modelAsString": false
+          }
+        },
+        "lastModifiedAt": {
+          "type": "string"
+        }
+      }
+    },
+    "InferenceResourceBase": {
+      "description": "Azure Resource Manager resource envelope for Inference objects.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "Specifies the resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Specifies the type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Specifies the name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "systemData": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/SystemData"
+            }
+          ],
+          "readOnly": true
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource.",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Specifies the location of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
Is it possible that we just inherit the generic resource definition from `machineLearningServices.json` to get the basic properties for an azure resource (`id`, `location`, `type`, etc)?